library/src/main/res/layout/mapbox_mapview_internal.xml

```diff
@@ -49,29 +50,62 @@
     <!-- This is the target button shown on the bottom-right -->
     <ImageButton
         android:id="@+id/ib_mapview_focusOnMyLocationIcon"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:src="@drawable/ic_my_location"
-        android:layout_marginBottom="20dp"
+        android:layout_width="70dp"
+        android:layout_height="70dp"
+        android:src="@drawable/ic_cross_hair"
+        android:layout_marginTop="20dp"
         android:layout_marginRight="20dp"
-        android:layout_gravity="bottom|right"
+        android:layout_gravity="top|right"
         android:padding="10dp"
         android:background="@drawable/circle_button"
-        android:layout_alignParentRight="true"
-        android:layout_alignParentBottom="true"
-        android:visibility="gone"/>
+        android:visibility="gone" />
 
     <ImageButton
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_gravity="bottom|right"
-        android:layout_margin="20dp"
-        android:id="@+id/imgBtn_mapview_locationAdditionBtn"
-        android:background="@drawable/circle_button_black_border"
-        android:src="@drawable/ic_add"
-        android:padding="10dp"
-        android:textColor="@color/white"
-        android:visibility="gone"/>
+        android:id="@+id/map_view_hamburger_menu"
+        android:layout_width="70dp"
+        android:layout_height="70dp"
+        android:src="@drawable/ic_hamburger_menu"
+        android:layout_marginTop="20dp"
+        android:layout_marginLeft="20dp"
+        android:layout_gravity="top|left"
+        android:background="@drawable/rounded_square_bg"/>
+
+    <LinearLayout
+        android:layout_height="500dp"
+        android:layout_width="match_parent"
+        android:layout_gravity="bottom|left">
+        <android.support.design.widget.CoordinatorLayout
+            android:layout_height="match_parent"
+            android:layout_width="wrap_content">
+            <android.support.v7.widget.CardView
```

Comment:
Could you provide an illustration on how to achieve this? The pushing `+` when this is enabled. It would be cool if this is clear how host applications are supposed to use this without having the `+` button overlaid by their custom views. Even better if it enables for dynamic views

Reply:
Illustration provided.

Use the `TEST CARD VIEW` button.